src/main/java/com/internet/shop/model/User.java

```diff
@@ -26,6 +27,14 @@ public User(Long id, String name, String login, String password) {
         this.password = password;
     }
 
+    public User(Long id, String name, String login, String password, byte[] salt) {
```

Comment:
I think there are too many constructors here already 
do you use all of them?

Reply:
Nope, I've already removed one of them but did not push it. I am going to push it without redundant constructor.